_guides/scheduling-flow-using-cron.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Scheduling a flow with CRON
+layout: article
+section: Building integration flows
+category: component
+order: 4
+since: 20181218
+---
+
+Scheduling tasks are part of the Elastic Platform's functionality. In this article, we are going to cover the scheduling scenario using CRON. 
+
+## What is CRON 
+
+[CRON](https://en.wikipedia.org/wiki/Cron) is the UNIX-like job scheduler that executes commands or scripts at given time and date. 
+CRON is driven by a *cron table*, to run periodically on a given schedule. See the *crontab* line on the screenshot below:
+
+![Crontab](/assets/img/developer-guide/cron/cron.png "Crontab")
+
+> **Please Note**: By default, our System repeats the flow's execution every 3 minutes. However, you are free to customize the time range according to your needs.
+
+The line's syntax expects a CRON expression made of five values. For more information on how to generate CRON's expressions, please follow this [link](https://www.freeformatter.com/cron-expression-generator-quartz.html).
+
+## The Scheduling
+
+Please be aware, that changing the flow's schedule means modifying the flow. Here some of the requirements to follow: 
+
+*   For modifying a flow, you should have specific permissions.
+*   As soon as you modify the flow's schedule, you should restart it to let the System apply the changes.
+
+To proceed further with CRON scheduling follow the steps below:
+
+**First step**
+
+Create a [flow](#realtime-flows) or pick the existing one by navigating to the **Workspace > Flows > Choose the necessary Flow > Settings** tab. See the screenshot below.
+
+![CRON Navigation](/assets/img/developer-guide/cron/cron_navigation.png "CRON Navigation")
+
+**Second step**
+
+The **Scheduling tab** consists of the **Cron Expression** section and the **Next Occurrences** one. 
+
+Specify the necessary value into the *Minute field* that is under the **Cron Expression** title. For example, we want to restart the flow every minute, so we type in the following `*/1` value. The **Next Occurrences** section represents the result. Check the screenshot below.
+
+![Scheduling](/assets/img/developer-guide/cron/scheduling.png "Scheduling")
```

Comment:
Please always make screenshot where you can see the entire application, not just the cropped area of the application. Showing the reader the entire application helps him to understand the context and where she is currently is.

Reply:
Screenshots were updated.